lib/Storage/BeeSwarm.php

```diff
@@ -427,9 +444,7 @@ public function getMetaData($path)
 
 	public function getDirectoryContent($path): \Traversable
 	{
-		$rows = $this->filemapper->getPathTree($path, $this->storageId,
-		                                       incSelf: false,
-		                                       recursive: false);
+		$rows = $this->filemapper->getPathTree($path, $this->storageId, false);
```

Comment:
@retrevor would you please verify this line change is related to your work? I think it's also still related to #68 and shouldn't be reverted